sdk/python/foundation-models/system/content-safety/deploy_with_content_safety.ipynb

```diff
@@ -0,0 +1,803 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to create an Safety-Enabled LLaMa Online Endpoint\n",
```

Comment:
I would call it "Content safety enabled" to align with ACS naming, and the label we put in AML Studio UI.

Reply:
IIRC this was the terminology the //build keynote kept using too. "Content safety..."

Reply:
The official product name is **Azure AI Content Safety**. Let's try to use the official name wherever possible.

Reply:
updated to Azure AI Content Safety.